common/remote_util.cc

```diff
@@ -47,10 +49,29 @@ void RemoteUtil::SetScpCommand(std::string scp_command) {
   scp_command_ = std::move(scp_command);
 }
 
+void RemoteUtil::SetSftpCommand(std::string sftp_command) {
+  sftp_command_ = std::move(sftp_command);
+}
+
 void RemoteUtil::SetSshCommand(std::string ssh_command) {
   ssh_command_ = std::move(ssh_command);
 }
 
+// static
+std::string RemoteUtil::ScpToSftpCommand(std::string scp_command) {
+  // "scp", "winscp.exe", "C:\path\to\scp", "/scppath/scp --foo" etc.
+  size_t pos = 0;
+  while ((pos = scp_command.find("scp", pos)) != std::string::npos) {
```

Comment:
string::find is case sensitive, but on Windows you probably want to ignore the case for the binary.

Reply:
Good point, thanks! Done.